fuzzing/common.bzl

```diff
@@ -15,18 +15,25 @@
 
 """This file contains common rules for fuzzing test."""
 
+load("@bazel_skylib//rules:common_settings.bzl", "BuildSettingInfo")
+
 def _fuzzing_launcher_impl(ctx):
     # Generate a script to launcher the fuzzing test.
     script = ctx.actions.declare_file("%s" % ctx.label.name)
+    args = []
+
+    if ctx.attr.is_regression:
+        args.append("--regression=True")
```

Comment:
I believe this still doesn't fully address the consistency issue: Some args are still part of the template, and others are constructed here. To give the reader a complete picture of the command line, consider building the args completely here (starting from the launcher_path, then the target binary path, etc.)